news/7959.removal

```diff
@@ -0,0 +1,17 @@
+Removes shebang from nonexecutable script.
```

Comment:
I think `removal` is used for a functionality that has been deprecated and removed. A change like this doesn't actually remove / break anything, the shebang had no purpose. As such, I'd consider going with `trivial`.

https://pip.pypa.io/en/latest/development/contributing/#choosing-the-type-of-news-entry